ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.html

```diff
@@ -33,17 +33,23 @@
 	<mat-button-toggle class="driver-btn" value="italic" (click)="toggleStateToDriver()">Driver</mat-button-toggle>
 </mat-button-toggle-group>
 <button
-	mat-raised-button
-	class="main-screen-action-btn save-route-btn"
-	(click)="saveRoute()"
-	*ngIf="isShownSaveRouteButton"
```

Comment:
matTooltip is not set for edit trip button + change color for  edit trip and add trip buttons

Reply:
fix